core/DependencyInjection/ContainerBuilder.php

```diff
@@ -198,4 +207,9 @@ private function getExtensionsManager(): common_ext_ExtensionsManager
     {
         return $this->legacyContainer->get(common_ext_ExtensionsManager::SERVICE_ID);
     }
+
+    private function isApplicationInstalled(): bool
+    {
+        return file_exists($this->configPath . 'generis/installation.conf.php');
```

Comment:
Suggestion just in case the `configPath` is provided without a trailing slash, so we are sure the Generis dir and `confiigPath` are separated by a slash.

```suggestion
        return file_exists(($this->configPath !== null ? rtrim($this->configPath, '/') : '.') . '/generis/installation.conf.php');
```

Reply:
Done, thanks!